### PR TITLE
phpmd test 1

### PR DESCRIPTION
**Pull request by clear-nishikubo**

### CODE REVIEW

File: app/Controller/PagesController.php [hunk 1]
```diff
@@ -45,6 +45,7 @@ class PagesController extends AppController {
  *	or MissingViewException in debug mode.
  */
 	public function display() {
+    $aaaaaaaaaa;
```
Comment (clear-nishikubo) [inline app/Controller/PagesController.php]: [development] [code_sniffer]

Comment (clear-nishikubo) [inline app/Controller/PagesController.php]: [development] [phpmd] Avoid unused local variables such as '$aaaaaaaaaa'.

File: app/Controller/PagesController.php [hunk 2]
```diff
@@ -45,6 +45,8 @@ class PagesController extends AppController {
...
```
Comment (clear-nishikubo) [inline app/Controller/PagesController.php]: [development] [phpmd] Avoid unused local variables such as '$aaaaaaaaaa'.

File: app/Controller/PagesController.php [hunk 3]
```diff
@@ -45,6 +45,8 @@ class PagesController extends AppController {
...
+    $bbbbbbbbbb;
```
Comment (clear-nishikubo) [inline app/Controller/PagesController.php]: [development] [phpmd] Avoid unused local variables such as '$bbbbbbbbbb'.

File: app/Controller/PagesController.php (hunk 2 above)
Comment (clear-nishikubo) [inline app/Controller/PagesController.php]: [development] [phpmd] Avoid unused local variables such as '$aaaaaaaaaa'.

Comment (clear-nishikubo) [inline app/Controller/PagesController.php]: [development] [phpmd] Avoid unused local variables such as '$aaaaaaaaaa'.

File: app/Controller/PagesController.php (hunk 3 above)
Comment (clear-nishikubo) [inline app/Controller/PagesController.php]: [development] [phpmd] Avoid unused local variables such as '$bbbbbbbbbb'.

File: app/Controller/PagesController.php (hunk 2 above)
Comment (clear-nishikubo) [inline app/Controller/PagesController.php]: [development] [phpmd] Avoid unused local variables such as '$aaaaaaaaaa'.

Comment (clear-nishikubo) [inline app/Controller/PagesController.php]: [development] [phpmd] Avoid unused local variables such as '$aaaaaaaaaa'.

Comment (clear-nishikubo) [inline app/Controller/PagesController.php]: [development] [phpmd] Avoid unused local variables such as '$aaaaaaaaaa'.

File: app/Controller/PagesController.php (hunk 3 above)
Comment (clear-nishikubo) [inline app/Controller/PagesController.php]: [development] [phpmd] Avoid unused local variables such as '$bbbbbbbbbb'.

Comment (clear-nishikubo) [inline app/Controller/PagesController.php]: [development] [phpmd] Avoid unused local variables such as '$bbbbbbbbbb'.

File: app/Controller/PagesController.php (hunk 2 above)
Comment (clear-nishikubo) [inline app/Controller/PagesController.php]: [development] [phpmd] Avoid unused local variables such as '$aaaaaaaaaa'.

File: app/Controller/PagesController.php (hunk 3 above)
Comment (clear-nishikubo) [inline app/Controller/PagesController.php]: [development] [phpmd] Avoid unused local variables such as '$bbbbbbbbbb'.

File: app/Controller/PagesController.php [hunk 4]
```diff
@@ -74,3 +115,44 @@ public function display() {
 		}
 	}
 }
+
+/**
+ * CyclomaticComplexity
+ */
+// Cyclomatic Complexity = 12
+class Foo {
+    public function example()  {
```
Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [phpmd] The method example() has a Cyclomatic Complexity of 11. The configured cyclomatic complexity threshold is 10.

File: app/Controller/PagesController.php [hunk 5]
```diff
@@ -45,6 +45,9 @@ class PagesController extends AppController {
  *	or MissingViewException in debug mode.
  */
 	public function display() {
+    $aaaaaaaaaa;
```
Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Line indented incorrectly; expected at least 8 spaces, found 4

File: app/Controller/PagesController.php [hunk 6]
```diff
@@ -45,6 +45,9 @@ class PagesController extends AppController {
...
+    $bbbbbbbbbb;
```
Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Line indented incorrectly; expected at least 8 spaces, found 4

File: app/Controller/PagesController.php [hunk 7]
```diff
@@ -45,6 +45,9 @@ class PagesController extends AppController {
...
+    $cccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc;
```
Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Line indented incorrectly; expected at least 8 spaces, found 4

File: app/Controller/PagesController.php [hunk 8]
```diff
@@ -38,6 +38,44 @@ class PagesController extends AppController {
 	public $uses = array();
 
 /**
+ * TooManyMethods
+*/
```
Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Missing @return tag in function comment

Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Expected 1 space(s) before asterisk; 0 found

File: app/Controller/PagesController.php [hunk 9]
```diff
@@ -38,6 +38,44 @@ class PagesController extends AppController {
...
+public function display1() {
```
Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Line indented incorrectly; expected at least 4 spaces, found 0

Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Opening brace should be on a new line

File: app/Controller/PagesController.php [hunk 10]
```diff
@@ -38,6 +38,44 @@ class PagesController extends AppController {
...
+}
+public function display2() {
```
Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Missing function doc comment

Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Opening brace should be on a new line

File: app/Controller/PagesController.php [hunk 11]
```diff
@@ -38,6 +38,44 @@ class PagesController extends AppController {
...
+}
+public function display3() {
```
Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Missing function doc comment

Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Opening brace should be on a new line

File: app/Controller/PagesController.php [hunk 12]
```diff
@@ -38,6 +38,44 @@ class PagesController extends AppController {
...
+}
+public function display4() {
```
Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Missing function doc comment

Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Opening brace should be on a new line

File: app/Controller/PagesController.php [hunk 13]
```diff
@@ -38,6 +38,44 @@ class PagesController extends AppController {
...
+}
+public function display5() {
```
Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Missing function doc comment

Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Opening brace should be on a new line

File: app/Controller/PagesController.php [hunk 14]
```diff
@@ -38,6 +38,44 @@ class PagesController extends AppController {
...
+}
+public function display6() {
```
Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Missing function doc comment

Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Opening brace should be on a new line

File: app/Controller/PagesController.php [hunk 15]
```diff
@@ -38,6 +38,44 @@ class PagesController extends AppController {
...
+}
+public function display7() {
```
Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Missing function doc comment

Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Opening brace should be on a new line

File: app/Controller/PagesController.php [hunk 16]
```diff
@@ -38,6 +38,44 @@ class PagesController extends AppController {
...
+}
+public function display8() {
```
Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Missing function doc comment

Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Opening brace should be on a new line

File: app/Controller/PagesController.php [hunk 17]
```diff
@@ -38,6 +38,44 @@ class PagesController extends AppController {
...
+}
+public function display9() {
```
Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Missing function doc comment

Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Opening brace should be on a new line

File: app/Controller/PagesController.php [hunk 18]
```diff
@@ -38,6 +38,44 @@ class PagesController extends AppController {
...
+}
+public function display10() {
```
Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Missing function doc comment

Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Opening brace should be on a new line

File: app/Controller/PagesController.php [hunk 19]
```diff
@@ -38,6 +38,44 @@ class PagesController extends AppController {
...
+}
+public function display11() {
```
Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Missing function doc comment

Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Opening brace should be on a new line

File: app/Controller/PagesController.php [hunk 20]
```diff
@@ -38,6 +38,44 @@ class PagesController extends AppController {
...
+}
+public function display12() {
```
Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Missing function doc comment

Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Opening brace should be on a new line

File: app/Controller/PagesController.php [hunk 21]
```diff
@@ -38,6 +38,44 @@ class PagesController extends AppController {
...
+}
+public function display13() {
```
Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Missing function doc comment

Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Opening brace should be on a new line

File: app/Controller/PagesController.php [hunk 22]
```diff
@@ -38,6 +38,44 @@ class PagesController extends AppController {
...
+}
+public function display14() {
```
Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Missing function doc comment

Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Opening brace should be on a new line

File: app/Controller/PagesController.php [hunk 23]
```diff
@@ -38,6 +38,44 @@ class PagesController extends AppController {
...
+}
+public function display15() {
```
Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Missing function doc comment

Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Opening brace should be on a new line

File: app/Controller/PagesController.php [hunk 24]
```diff
@@ -38,6 +38,44 @@ class PagesController extends AppController {
...
+}
+public function display16() {
```
Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Missing function doc comment

Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Opening brace should be on a new line

File: app/Controller/PagesController.php [hunk 25]
```diff
@@ -38,6 +38,44 @@ class PagesController extends AppController {
...
+}
+public function display17() {
```
Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Missing function doc comment

Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Opening brace should be on a new line

File: app/Controller/PagesController.php [hunk 26]
```diff
@@ -74,3 +115,44 @@ public function display() {
 		}
 	}
 }
+
+/**
+ * CyclomaticComplexity
+ */
+// Cyclomatic Complexity = 12
+class Foo {
```
Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Each class must be in a file by itself

Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Each class must be in a namespace of at least one level (a top-level vendor name)

Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Opening brace of a class must be on the line after the definition

File: app/Controller/PagesController.php (hunk 4 above)
Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Missing function doc comment

Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Double space found

Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Opening brace should be on a new line

File: app/Controller/PagesController.php [hunk 27]
```diff
@@ -74,3 +115,44 @@ public function display() {
...
+    public function example()  {
+        if ($a == $b)  {
```
Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Expected "if (...) {\n"; found "if (...)  {\n"

Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Expected 1 space after closing parenthesis; found "  "

Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Double space found

File: app/Controller/PagesController.php [hunk 28]
```diff
@@ -74,3 +115,44 @@ public function display() {
...
+            if ($a1 == $b1) {
+                fiddle();
+            } else if ($a2 == $b2) {
```
Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Usage of ELSE IF is discouraged; use ELSEIF instead

Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Usage of ELSE IF not allowed; use ELSEIF instead

File: app/Controller/PagesController.php [hunk 29]
```diff
@@ -74,3 +115,44 @@ public function display() {
...
+                fiddle();
+            }  else {
```
Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Expected 1 space after closing brace; 2 found

Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Double space found

Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Expected "} else {\n"; found "}  else {\n"

File: app/Controller/PagesController.php [hunk 30]
```diff
@@ -74,3 +115,44 @@ public function display() {
...
+                fiddle();
+            }
+        } else if ($c == $d) {
```
Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Usage of ELSE IF is discouraged; use ELSEIF instead

Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Usage of ELSE IF not allowed; use ELSEIF instead

File: app/Controller/PagesController.php [hunk 31]
```diff
@@ -74,3 +115,44 @@ public function display() {
...
+            while ($c == $d) {
+                fiddle();
+            }
+         } else if ($e == $f) {
```
Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Line indented incorrectly; expected 8 spaces, found 9

Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Closing brace indented incorrectly; expected 8 spaces, found 9

Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Usage of ELSE IF is discouraged; use ELSEIF instead

Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Usage of ELSE IF not allowed; use ELSEIF instead

File: app/Controller/PagesController.php [hunk 32]
```diff
@@ -74,3 +115,44 @@ public function display() {
...
+            for ($n = 0; $n < $h; $n++) {
+                fiddle();
+            }
+        } else{
```
Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Line indented incorrectly; expected 12 spaces, found 8

Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Closing brace indented incorrectly; expected 9 spaces, found 8

Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Expected 1 space after ELSE keyword; 0 found

Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Expected "} else {\n"; found "} else{\n"

File: app/Controller/PagesController.php [hunk 33]
```diff
@@ -74,3 +115,44 @@ public function display() {
...
+            switch ($z) {
```
Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Line indented incorrectly; expected 16 spaces, found 12

File: app/Controller/PagesController.php [hunk 34]
```diff
@@ -74,3 +115,44 @@ public function display() {
...
+                case 1:
```
Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Line indented incorrectly; expected 20 spaces, found 16

File: app/Controller/PagesController.php [hunk 35]
```diff
@@ -74,3 +115,44 @@ public function display() {
...
+                    fiddle();
```
Comment (sumyapp) [inline app/Controller/PagesController.php]: [staging] [code_sniffer] Line indented incorrectly; expected at least 24 spaces, found 20

File: app/Controller/PagesController.php (hunk 1 above)
Comment (clear-nishikubo) [inline app/Controller/PagesController.php]: [development] [phpmd] Avoid unused local variables such as '$aaaaaaaaaa'.

Comment (clear-nishikubo) [inline app/Controller/PagesController.php]: [development] [phpmd] Avoid unused local variables such as '$aaaaaaaaaa'.

Comment (clear-nishikubo) [inline app/Controller/PagesController.php]: [development] [phpmd] Avoid unused local variables such as '$aaaaaaaaaa'.

Comment (clear-nishikubo) [inline app/Controller/PagesController.php]: [development] [phpmd] Avoid unused local variables such as '$aaaaaaaaaa'.

Comment (clear-nishikubo) [inline app/Controller/PagesController.php]: [development] [phpmd] Avoid unused local variables such as '$aaaaaaaaaa'.

**Comment by sumyapp** [comment]
テストスクリプトで使っているようなためReopen

**Pull request closed without merging**